packages/@react-spectrum/color/docs/ColorSwatch.mdx

```diff
@@ -0,0 +1,121 @@
+{/* Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import colorTypes from 'docs:@react-types/color/src/index.d.ts';
+import docs from 'docs:@react-spectrum/color';
+import {HeaderInfo, PropTable, TypeLink, PageDescription} from '@react-spectrum/docs';
+import {Layout} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/color/package.json';
+import statelyDocs from 'docs:@react-stately/color';
+
+export default Layout;
+
+```jsx import
+import {ColorSwatch} from '@react-spectrum/color';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Color
+keywords: [color swatch]
+---
+
+# ColorSwatch
+
+<PageDescription>{docs.exports.ColorSwatch.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ColorSwatch']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/swatch/'}
+  ]} />
```

Comment:
I think that should be the case already? It is in a pre-release package so the import is already the individual package.

Reply:
Ah sorry, not sure where the rest of the comment went when I was editing it, for context:
Will ColorSwatch/ColorPicker/etc all hit GA at the same time as ColorArea/ColorWheel/etc? If not then we will have a similar situation to SubMenu where the menu package itself is fully released but there are components within that aren't in the monopackage due to their alpha state relative to the rest of the package. If that is the case, we'll want to make sure imports referencing those alpha components in the docs are still pointing to the specific package instead of the monopackage, otherwise people will probably copy paste the monopackage import and wonder why it doesn't work.

Reply:
oh yeah, true. I think they will probably hit GA at the same time but we'll see. In my other PR I added a `preRelease` flag to the docs so pre-release components in the mono package can be marked with a badge still. We could use something similar to inform the import rewriter.